instrumentation-api-appender/src/main/java/io/opentelemetry/instrumentation/api/appender/LogEmitter.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.appender;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * A {@link LogEmitter} is the entry point into a log pipeline.
+ *
+ * <p>Obtain a log builder via {@link #logBuilder()}, add properties using the setters, and emit it
+ * via {@link LogBuilder#emit()}.
+ */
+@ThreadSafe
+public interface LogEmitter {
+
+  /** Return a {@link LogBuilder} to emit a log. */
```

Comment:
```suggestion
  /** Return a new {@link LogBuilder} to emit a log. */
```

Nitpicky, but I think calling out that it's a new instance helps the confusing api a little.